zman-frontend/src/main/frontend/src/templates/user/user.tpl.html

```diff
@@ -0,0 +1,7 @@
+<div ng-init="currentUserCtrl.getCurrentUser()">
+    <div ng-if="currentUserCtrl.currentUser != 'anonymousUser'">
```

Comment:
maybe add something like "isLoggedIn" checks ? using string constant to identify not authenticated users doesn't look pretty good imho